buildSrc/src/main/kotlin/io/spine/internal/gradle/github/pages/Update.kt

```diff
@@ -53,52 +53,52 @@ private class Operation(
     private val rootFolder: File,
     checkoutTempFolder: Path,
     private val javadocOutputPath: Path,
+    private val dokkaOutputPath: Path,
     private val logger: Logger
 ) {
 
     private val ghRepoFolder: File = File("${checkoutTempFolder}/${Branch.ghPages}")
-    private val docDirPostfix = "reference/$project.name"
-    private val mostRecentDocDir = File("$ghRepoFolder/$docDirPostfix")
+
+    private val javadocDirPostfix = "reference/${project.name}"
```

Comment:
Let's have two classes each dedicated to updating its own type of our documentation. 

Right now the issues detected in this type rightfully tell you there's already too much of responsibility here.